services/web/server/src/simcore_service_webserver/invitations/_db.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+import sqlalchemy as sa
+from aiohttp import web
+from models_library.users import GroupID
+from simcore_postgres_database.models.groups import user_to_groups
+from simcore_postgres_database.models.users import users
+
+from ..db.plugin import get_database_engine
+
+_logger = logging.getLogger(__name__)
+
+
+async def is_user_registered_in_product(
+    app: web.Application, email: str, product_group_id: GroupID
```

Comment:
Q: isn't it purely from the DB layer this one the same as above:
`services/web/server/src/simcore_service_webserver/groups/_db.py`?
The only thing that changed is the name of the function and parameter for this special use case, which points to me that the "naming" should be in the "Service layer", but the DB function might be used the same one.

Reply:
totally agree! thx!